.github/workflows/pr.yaml

```diff
@@ -8,9 +8,9 @@ jobs:
     steps:
 
     - name: Set up Go
-      uses: actions/setup-go@v2
+      uses: actions/setup-go@v5
       with:
-        go-version: '1.22'
+        go-version-file: go.mod
```

Comment:
The reason for the go version not being found should be the `setup-go` step should be put behind the `checkout` step.

Reply:
Good catch, will fix this right now!